js/app/interfaces/card.js

```diff
@@ -302,18 +299,18 @@ const Card = new Lang.Interface({
         if (!this.model.thumbnail_uri)
             return;
 
-        let coveredFrame = new ImageCoverFrame.ImageCoverFrame();
-        frame.add(coveredFrame);
-
-        let cancellable = null;
-        let file = Gio.File.new_for_uri(this.model.thumbnail_uri);
-        let stream = file.read(cancellable);
-        coveredFrame.set_content(stream);
-        frame.visible = true;
+        let bg_provider = new Gtk.CssProvider();
+        bg_provider.load_from_data('frame {\
+            background-size: cover;\
```

Comment:
Good point, I should probably leave this commit off and open a separate issue for it, to be merged after we get GSK.